### ISSUE TITLE
Clothes Washer: Door Close?

### ISSUE THREAD

**Issue by Jeppedy** [ollo69/ha-smartthinq-sensors#365]
What is the Door Close attribute on the Clothes Washer.  Mine always reads Off, and never changes.

**Comment by ollo69** [comment]
Door Close should represent the status of the Washer machine door. Attribute should change based on the state of the door. If does not change please attach integration diagnostic so that I can check the payload provided by the device. Please attach diagnostic that represent both status (machine with door closed and open)

**Comment by ollo69** [comment]
Any feedback on this? Can we close?

**Comment by petervk** [comment]
Hi, I have this same issue. The Washer Door Close sensor never changes. Here are 4 diagnostic downloads, I believe that the first two were when the door was closed and the second two are when the door was open. I did a Ctrl-F on these files and don't see any difference in anything door related.
[smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-1.json.txt](https://github.com/ollo69/ha-smartthinq-sensors/files/9736154/smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-1.json.txt)
[smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-2.json.txt](https://github.com/ollo69/ha-smartthinq-sensors/files/9736155/smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-2.json.txt)
[smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-3.json.txt](https://github.com/ollo69/ha-smartthinq-sensors/files/9736156/smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-3.json.txt)
[smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-4.json.txt](https://github.com/ollo69/ha-smartthinq-sensors/files/9736157/smartthinq_sensors-a3f35ca73f862c5e352fb08fb4c72454-Washer-d2c8d821dc6f0b0365157d5c8bc74306-4.json.txt)
